### PR TITLE
Reverse connected components labeling

### PR DESCRIPTION
**Pull request by cquammen**
Flip the labels so that the largest component has the largest label
value and the smallest component has the smallest value. The label
value is really arbitrary, so changing the labeling in this way
shouldn't be a big deal. It does, however, make display of the largest
components via color map nicer because the largest component now has a
value most different from the background label.

### CODE REVIEW

**Comment by cquammen** [comment]
Mind reviewing @cjh1 @mathturtle @cryos ? Thanks.

**Comment by cquammen** [comment]
First implementation was terribly memory inefficient. Now uses in-place operations instead.

File: tomviz/python/ConnectedComponents.py [hunk 1]
```diff
@@ -72,9 +72,23 @@ def transform_scalars(dataset):
...
+        maximum = np.max(label_buffer)
+
+        # Try more memory-efficient approach
+        gt_zero = label_buffer > 0
+        label_buffer[gt_zero] = minimum - label_buffer[gt_zero] + maximum
+
         label_map_data_set = vtk.vtkImageData()
         label_map_data_set.CopyStructure(dataset)
 
+        #utils.set_label_map(label_map_data_set, flipped_label_buffer)
```
Comment (cjh1) [inline tomviz/python/ConnectedComponents.py]: Should we remove this?

Comment (cquammen) [inline tomviz/python/ConnectedComponents.py]: Indeed. Thanks for catching.

**Comment by cjh1** [comment]
Had to break out ipython to understand the numpy array operations but LGTM :smile:

**Pull request merged**